src/collar/oc_anim.lsl

```diff
@@ -93,778 +97,785 @@ list g_lMenuIDs; // Three strided list of kAvatar, kDialogID, sMenuName
 }
 */
 
 Dialog(key kID, string sPrompt, list lChoices, list lUtilityButtons, integer iPage, integer iAuth, string sName) {
-  key kMenuID = llGenerateKey();
-  llMessageLinked(LINK_DIALOG, DIALOG, (string)kID + "|" + sPrompt + "|" + (string)iPage + "|" + llDumpList2String(lChoices, "`") + "|" + llDumpList2String(lUtilityButtons, "`") + "|" + (string)iAuth, kMenuID);
-
-  integer iIndex = llListFindList(g_lMenuIDs, [kID]);
-  if (~iIndex) {
-    g_lMenuIDs = llListReplaceList(g_lMenuIDs, [kID, kMenuID, sName], iIndex, iIndex + g_iMenuStride - 1);
-  } else {
-    g_lMenuIDs += [kID, kMenuID, sName];  // We've not already given this user a menu. append to list
-  }
-  //Debug("Made " + sName + " menu.");
+    key kMenuID = llGenerateKey();
+    llMessageLinked(LINK_DIALOG, DIALOG, (string)kID + "|" + sPrompt + "|" + (string)iPage + "|" + llDumpList2String(lChoices, "`") + "|" + llDumpList2String(lUtilityButtons, "`") + "|" + (string)iAuth, kMenuID);
+
+    integer iIndex = llListFindList(g_lMenuIDs, [kID]);
+    if (~iIndex) {
+        g_lMenuIDs = llListReplaceList(g_lMenuIDs, [kID, kMenuID, sName], iIndex, iIndex + g_iMenuStride - 1);
+    } else {
+        g_lMenuIDs += [kID, kMenuID, sName];  // We've not already given this user a menu. append to list
+    }
+    //Debug("Made " + sName + " menu.");
 }
 
 string Checkbox(integer iValue, string sLabel) {
-  if (iValue) return "☑ " + sLabel;
-  else return "☐ " + sLabel;
+    if (iValue) return "☑ " + sLabel;
+    else return "☐ " + sLabel;
 }
 
 AnimMenu(key kID, integer iAuth) {
-  string sPrompt = "\n[Animations]\n\n%WEARERNAME%";
-  list lButtons;
-
-  if (g_bAnimLock) {
-    sPrompt += " is forbidden to change or stop poses on their own";
-    lButtons = ["☑ AnimLock"];
-  } else {
-    sPrompt += " is allowed to change or stop poses on their own";
-    lButtons = ["☐ AnimLock"];
-  }
-
-  if (llGetInventoryType("~stiff") == INVENTORY_ANIMATION) {
-    if (g_bPosture) {
-      sPrompt += " and has their neck forced stiff.";
-      lButtons += ["☑ Posture"];
+    string sPrompt = "\n[Animations]\n\n%WEARERNAME%";
+    list lButtons;
+
+    if (g_bAnimLock) {
+        sPrompt += " is forbidden to change or stop poses on their own";
+        lButtons = ["☑ AnimLock"];
     } else {
-      sPrompt += " and can relax their neck.";
-      lButtons += ["☐ Posture"];
+        sPrompt += " is allowed to change or stop poses on their own";
+        lButtons = ["☐ AnimLock"];
     }
-  }
 
-  lButtons += Checkbox(g_bTweakPoseAO, "AntiSlide");
-  lButtons += ["AO Menu", "AO ON", "AO OFF", "Pose"];
+    if (llGetInventoryType("~stiff") == INVENTORY_ANIMATION) {
+        if (g_bPosture) {
+            sPrompt += " and has their neck forced stiff.";
+            lButtons += ["☑ Posture"];
+        } else {
+            sPrompt += " and can relax their neck.";
+            lButtons += ["☐ Posture"];
```

Comment:
I like how you moved the posture option into a inventory check. Though i still need to check that this actually works as intended. Will update my review after testing

Reply:
Ummm, but i haven't moved anything here. Posture option was already inside inventory check, isn't it?

Reply:
Ah it looks like you did the way this git diff came out...